wemake_python_styleguide/violations/consistency.py

```diff
@@ -2062,3 +2064,33 @@ class BracketBlankLineViolation(TokenizeViolation):
 
     error_template = 'Found an unnecessary blank line before a bracket'
     code = 355
+
+
+@final
+class IterableUnpackingViolation(ASTViolation):
+    """
+    Forbids unneccessary iterable unpacking.
+
+    Reasoning:
+        We do this for consistency.
+
+    Solution:
+        Do not use iterables unpacking, when it's not neccessary.
+
+    Example::
+
+        # Correct:
+        [1, *numbers, 99]
+        {*iterable, *other_iterable}
+        first, *iterable = other_iterable
+
+        # Wrong:
+        [*iterable]
```

Comment:
I guess we should also show how to handle this situation. Let's add `list(iterable)` to the correct examples.